mysql/README.md

```diff
@@ -163,6 +163,9 @@ See also https://dev.mysql.com/doc/refman/5.7/en/environment-variables.html for
 
 This variable is mandatory and specifies the password that will be set for the MySQL `root` superuser account. In the above example, it was set to `my-secret-pw`.
 
+### `MYSQL_ROOT_HOST`
+This variable is optional. It allows you to use root connections from outside the container (for example from your host or remote machine). To allow root connections from your host machine set this value to `host.docker.internal`. The variable accepts only one entry, but wildcards are allowed (for example, `MYSQL_ROOT_HOST=172.*.*.*` or `MYSQL_ROOT_HOST=%`). If the variable is set a second root account will be created for the value you used, so you'll have `'root'@'localhost'` and `'root'@'${MYSQL_ROOT_HOST}'`.
```

Comment:
`host.docker.internal` will not work in the default configuration since we [set `skip-name-resolve`](https://github.com/docker-library/mysql/blob/d60655b1b42f677c315d5fe2ca0e87126ca921f5/8.0/config/conf.d/docker.cnf#L2):

>```console
>[Warning] [MY-010315] [Server] 'user' entry 'root@host.docker.internal' ignored in --skip-name-resolve mode.
>```